util/flags.py

```diff
@@ -100,7 +99,14 @@ def create_flags():
     f.DEFINE_string('checkpoint_dir', '', 'directory in which checkpoints are stored - defaults to directory "deepspeech/checkpoints" within user\'s data home specified by the XDG Base Directory Specification')
     f.DEFINE_integer('checkpoint_secs', 600, 'checkpoint saving interval in seconds')
     f.DEFINE_integer('max_to_keep', 5, 'number of checkpoint files to keep - default value is 5')
-    f.DEFINE_string('load', 'auto', '"last" for loading most recent epoch checkpoint, "best" for loading best validated checkpoint, "init" for initializing a fresh model, "auto" for trying the other options in order last > best > init')
+    f.DEFINE_string('load', 'auto', '"last" for loading most recent epoch checkpoint, "best" for loading best validated checkpoint, "init" for initializing a fresh model, "transfer" for transfer learning, "auto" for trying the other options in order last > best > init')
+
+    # Transfer Learning
+    
+    f.DEFINE_string ('source_model_checkpoint_dir', '', 'directory in which checkpoints are stored - defaults to directory "deepspeech/checkpoints" within user\'s data home specified by the XDG Base Directory Specification')
```

Comment:
I think, given your explanations, that it makes sense we keep this flag but I don't see it anymore ?

Reply:
This is the biggest fix to be made atm

Reply:
now `--load_checkpoint_dir`